lessons/introR/main.Rmd

```diff
@@ -294,3 +294,11 @@ library(knitr)
 knit('main.Rmd') ## or can use knitr::knit('main.Rmd')
 
 ```
+
+# Challenge: Try this out for yourself! #
+
+Make a table with the means of Agriculture, Examination, Education,
+and Infant.Mortality by Fertility as a factor that is split in the
```

Comment:
I'm just a bit confused by the "by Fertility". Do you mean "sorted by", "split by", "labelled by", etc.?